### ISSUE TITLE
DOI link is blank

### ISSUE THREAD

**Issue by benaltair** [bahaistudies/jbs#52]
![image](https://user-images.githubusercontent.com/17771229/116308098-f50c0400-a75b-11eb-808b-5ffbef28cf4f.png)
Reads `blank` on screen reader, text is missing.
